daemon-tests/tests/main/collaborative_settlement.rs

```diff
@@ -113,4 +120,66 @@ async fn collaboratively_close_an_open_cfd(
 
     confirm!(close transaction, order_id, maker, taker);
     wait_next_state!(order_id, maker, taker, CfdState::Closed);
+    verify_closed_cfds(&mut maker, &mut taker);
+}
+
+/// Sanity check the values we publish in projection
+fn verify_closed_cfds(maker: &mut Maker, taker: &mut Taker) {
+    assert_eq!(
+        maker.first_cfd().closing_price.unwrap(),
+        taker.first_cfd().closing_price.unwrap()
+    );
+    assert_eq!(
+        maker.first_cfd().contract_symbol,
+        taker.first_cfd().contract_symbol
+    );
+    assert_ne!(maker.first_cfd().position, taker.first_cfd().position);
+
+    assert_eq!(
+        maker.first_cfd().closing_price.unwrap(),
+        initial_price_for(maker.first_cfd().contract_symbol)
+    );
+
+    assert_maker_liquidation_price(maker);
+    assert_taker_liquidation_price(taker);
+}
+
+fn assert_maker_liquidation_price(maker: &mut Maker) {
+    assert_eq!(dummy_btc_price(), dec!(50_000), "precondition");
+    assert_eq!(dummy_eth_price(), dec!(1_500), "precondition");
+
+    let maker_position = maker.first_cfd().position;
+    let symbol = maker.first_cfd().contract_symbol;
+    let expected_liquidation_price = match (symbol, maker_position) {
+        // inverse payout curve
+        (ContractSymbol::BtcUsd, Position::Long) => dec!(25_000),
+        (ContractSymbol::BtcUsd, Position::Short) => dec!(21_000_000), // INFINITE
+        // quanto linear payout curve
+        (ContractSymbol::EthUsd, Position::Long) => dec!(1),
+        (ContractSymbol::EthUsd, Position::Short) => dec!(3_000),
+    };
+    assert_eq!(
+        maker.first_cfd().liquidation_price,
+        Price::new(expected_liquidation_price).unwrap()
+    );
```

Comment:
There is your bug: the problem is that the offer is from the point of view of the maker but in the frontend we need to know the liquidation price for the taker. See https://github.com/itchysats/itchysats/issues/2760#issuecomment-1232383910